### PR TITLE
Optionally resolve DNS

### PR DESCRIPTION
**Pull request by pquerna**

### CODE REVIEW

File: lib/mtr.js [hunk 1]
```diff
@@ -81,7 +81,10 @@ Mtr.prototype._run = function(target) {
     args.push('-6');
   }
 
-  args.push('--no-dns');
+  if (!this._options.resolve_dns) {
```
Comment (Kami) [inline lib/mtr.js]: For consistency please use `resolveDns` and the same thing I did with `cycleCount`.

File: tests/test-traceroute.js [hunk 2]
```diff
@@ -151,3 +151,23 @@ exports['test_traceroute_route_2'] = function(test, assert) {
     test.finish();
   });
 };
+
+exports['test_traceroute_route_with_hostnames'] = function(test, assert) {
```
Comment (Kami) [inline tests/test-traceroute.js:155]: I know the tests use fixtures, but to reduce the confusion, please update existing tests to pass `resolveDns: false` attribute in the options object to the constructor.

Comment (pquerna) [inline tests/test-traceroute.js:155]: isn't it the other way around? this one shoudl pass `resolveDns: true`

ResolveDns defaults to false?

Comment (Kami) [inline tests/test-traceroute.js:155]: Oh, yeah you are right, it defaults to false.

**Comment by Kami** [comment]
Merged, thanks.

**Comment by Kami** [comment]
v0.1.1 published to npm

**Pull request merged**